example/general_L10n_SG.md

```diff
@@ -0,0 +1,131 @@
+
+## Meaning-based translation
+
+When it comes to translation, meaning is everything. A translator needs to understand the source text's meaning exactly. You then find its most closely linked equivalent in your own language, without adding or subtracting meaning in your translation. Finding meaning-based equivalents between languages can be difficult. To help concentrate your thoughts, ask yourself questions like:
+
+* What does this word/sentence/string mean in English?
+* What is the message the author is trying to send?
+* How would I express that meaning in my own language?
+
+Sometimes translation memory and machine translation tools can offer bad suggestions for a translation. If you use either as part of your translation workflow, make sure to correct the suggestions before submitting them. Avoid literal translation at all costs. Watch out for words that might sound or look the same between English and your language, but have a different meaning.
+
+### Translating culture-specific references
+
+At times there will be English content included in Mozilla products or web projects (e.g., marketing campaigns) that makes references to American culture and concepts. When translating these, it is best to find an equivalent cultural reference within your own culture that accurately conveys the meaning of the English reference. For example, an American might say, `Good job, home run!` A `home run` is a baseball reference for a successful outcome. An appropriate translation would be an equivalent metaphor within your culture. Using soccer as an example, you might translate `Good job, home run!` into `Good job, nice goal!` in your language.
+
+### Units and measurement
+
+The translation should strive to achieve proper unit conversions for currency, measurements, etc. for the target audience. Source strings will use the imperial system (e.g., miles, pounds, feet, gallons, etc.). Target translations should convert imperial metrics to their measurement system.
+
+## Terminology
+
+You should be consistent in the use of existing reliable appropriate term bases in your language. These term bases could be developed and approved by the community, or leveraged from another party that adhere to national, international or local standards for software and internet terminology. Avoid inconsistent use of terminology within the project, within term base, or across all of your projects and using terminology from another subject matter (e.g., don't use medical terminology in Firefox).
+
+[Pontoon](https://pontoon.mozilla.org) features terminology support for all locales. Terminology is managed by each community in Pontoon itself. Here are a few things to keep in mind when managing your team's terminology:
+
+* Avoid overly borrowing English expressions
```

Comment:
List items should end with a period, if we want to keep using our internal style guide ;-)